src/views/CharacterListView.js

```diff
@@ -1,21 +1,21 @@
 import React from "react";
 import { connect } from "react-redux";
-
+import Loader from 'react-loader-spinner'
 import { CharacterList } from "../components";
-// import actions
+import { getCharacters } from '../actions'
 
 class CharacterListView extends React.Component {
   constructor() {
     super();
   }
 
   componentDidMount() {
-    // call our action
+    this.props.getCharacters();
   }
 
   render() {
     if (this.props.fetching) {
-      // return something here to indicate that you are fetching data
+      <Loader type="Circles" color="#424242" height={80} width={80}/>
```

Comment:
```suggestion
     return <Loader type="Circles" color="#424242" height={80} width={80}/>
```
You have to make sure to return something for it to display